Src/Version.cs

```diff
@@ -62,5 +62,10 @@ public Version(int major, int minor, int patch)
         /// A shorthand for writing <c>new Version(0, 0, 0)</c>.
         /// </summary>
         public static Version Zero => new Version(0, 0, 0);
+
+        public static implicit operator int(Version version)
```

Comment:
Where is this implicit conversion useful?

Reply:
Doesn't hurt and for debug purpose is good